root/static/styles/edit.less

```
@@ -164,6 +164,16 @@ table.vote-tally {
         }
     }
 
+    h3.Approve {
+        a img {
+            border-color: @yes-border;
+        }
+
+        .voting-icon {
+            background-image: data-uri('../images/icons/tickextrasharp.png');
```

should be tick.png right?
```suggestion
            background-image: data-uri('../images/icons/tick.png');
```

---

It really should, yes. Sigh. Thanks!